apps/omg/test/omg/state/persistence_test.exs

```diff
@@ -198,7 +198,10 @@ defmodule OMG.State.PersistenceTest do
 
   defp persist_common(state, db_updates, db_pid) do
     assert :ok = OMG.DB.multi_update(db_updates, db_pid)
-    assert state == state_from(db_pid)
+    assert drop_recently_spent(state) == state_from(db_pid)
     state
   end
+
+  # NOTE: `recently_spent` is in-memory property of the %Core{}, which isn't persisted so it can't be recovered from db
+  defp drop_recently_spent(state), do: %Core{state | recently_spent: []}
```

Comment:
eh, was going to comment to not reach into internals, but in fact this whole test is a bit silly right now. Let's stop and think what to do with it.